### PR TITLE
iOS UI

### PR DESCRIPTION
**Pull request by abeshi-softwire**
- Date formatted differently (now dates that are "today" display as `HH:mm (in xx hr/min)`
  - Added fallback dateFormatter that does the old `HH:mm dd/MM/yy` - removed seconds
- Corrected station label placement and content
- Added a header
- Added title row to table

### CODE REVIEW

File: native/KotlinIOS/KotlinIOS/JourneyCell.swift [hunk 1]
```diff
@@ -14,18 +14,33 @@ class JourneyCell: UITableViewCell {
...
+    
+    func toHumanReadableDate(_ date: Date) -> String {
+        // If today: HH:mm (in xx min/hr)
+        let hourFormatter = DateFormatter()
+        hourFormatter.dateFormat = "HH:mm"
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "HH:mm dd/MM/yy"
+        
+        let now = Date()
+        if (Calendar.current.isDate(date, inSameDayAs: now)) {
```
Comment (thevirtuoso1973) [inline native/KotlinIOS/KotlinIOS/JourneyCell.swift]: I think the user would find this useful even if the date is _not_ in the same day. E.g. it's currently almost midnight and the departure time is 3 hours way, which is 'the next day' but still only 3 hours away.

Comment (abeshi-softwire) [reply native/KotlinIOS/KotlinIOS/JourneyCell.swift]: That's a fair point. What would you class as close enough to show hours rather than a date?

Comment (thevirtuoso1973) [reply native/KotlinIOS/KotlinIOS/JourneyCell.swift]: For times that are up to 12 hours away. Beyond that, hours are probably less useful than a time and date.

Comment (abeshi-softwire) [reply native/KotlinIOS/KotlinIOS/JourneyCell.swift]: Alright, I'll fix that

**Review by thevirtuoso1973 — APPROVED** [review]

**Pull request merged**